perma_web/perma/settings/deployments/settings_common.py

```diff
@@ -465,3 +470,20 @@ def _pywb_static_dir_location():
 # If technical problems prevent proper analysis of a capture,
 # should we default to private?
 PRIVATE_LINKS_ON_FAILURE = False
+
+
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': (
+        'api2.authentication.TokenAuthentication',  # authenticate with ApiKey token
+        'rest_framework.authentication.SessionAuthentication',  # authenticate with Django login
+    ),
```

Comment:
This simple config is unacceptable. I demand we return to something in the style of https://github.com/harvard-lil/perma/blob/develop/perma_web/api/authentication.py. For pity's sake.